### ISSUE TITLE
Application not working inside the Drupal VM (Ubuntu 16.04.3 LTS)

### ISSUE THREAD

**Issue by msha86** [geerlingguy/drupal-vm#1627]
### Issue Type

 - Bug Report / Support Request

### Your Environment

$ vagrant --version
Vagrant 2.0.1
```
ansible 2.4.1.0
  config file = None
  configured module search path = [u'/home/vagrant/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.12 (default, Nov 19 2016, 06:48:10) [GCC 5.4.0 20160609]

```

### Your OS

 - **Windows** (version)

### Full console output

[Wed Nov 29 18:54:45.590113 2017] [proxy_fcgi:error] [pid 2789:tid 139675309233920] [client 192.168.137.1:63536] Premature end of script headers: index.php
[Wed Nov 29 18:54:45.590170 2017] [proxy_fcgi:error] [pid 2789:tid 139675309233920] [client 192.168.137.1:63536] AH01070: Error parsing script headers
[Wed Nov 29 18:54:45.590173 2017] [proxy_fcgi:error] [pid 2789:tid 139675309233920] (22)Invalid argument: [client 192.168.137.1:63536] AH01075: Error dispatching request to :

### Summary
I am getting the issue running the local drupal 8 environment. The issue we have identify that inside the file "development.services.yml" debug cacheability header set true (http.response.debug_cacheability_headers: true). When we set this to false . Environment start working. The error reported in apache error log is:

[Wed Nov 29 18:54:45.590113 2017] [proxy_fcgi:error] [pid 2789:tid 139675309233920] [client 192.168.137.1:63536] Premature end of script headers: index.php
[Wed Nov 29 18:54:45.590170 2017] [proxy_fcgi:error] [pid 2789:tid 139675309233920] [client 192.168.137.1:63536] AH01070: Error parsing script headers
[Wed Nov 29 18:54:45.590173 2017] [proxy_fcgi:error] [pid 2789:tid 139675309233920] (22)Invalid argument: [client 192.168.137.1:63536] AH01075: Error dispatching request to :
In our findings its related to the header limit in the Apache/FastCGI configuration . I try to increase the LimitRequestLine ,LimitRequestFieldSize in apache config but its not working.

I am using the ubuntu based Drupal VM
Distributor ID: Ubuntu
Description: Ubuntu 16.04.3 LTS
Release: 16.04
Codename: xenial

**Comment by geerlingguy** [comment]
@msha86 - This looks like it's an application issue—something to do with your Drupal site codebase.

```
Premature end of script headers: index.php
AH01070: Error parsing script headers
```

Judging by these errors, I'm guessing it is something to do with whitespace after a PHP closing tag, or a syntax error in one of the PHP files.

I would recommend asking about these errors or researching more on sites like Stack Exchange, or Drupal Answers, as they don't look related to Drupal VM or the underlying webserver setup.

**Comment by VinceJones** [comment]
I was experiencing this same issue. I tried changing the http.response.debug_cacheability_headers to false and the issue was solved. I'm going to try and dig further in and see what the underlying issue might be. I am using Acquia BLT with Drupal VM using a Lightning profile for additional context.

**Comment by cosmicdreams** [comment]
Hey @VinceJones, talking about his with @geerlingguy at Midcamp.  Things to check (in order):

1. Does this happen on Mac too?
2. Is this the result of htpassword
3. Try destroying your VM, updating your box, and rebuilding the VM.

**Comment by TheThor** [comment]
I'm also having this problem. @VinceJones can you further explain how you remembered to disable the headers cache? I've been having an apache error **"Premature end of script headers: index.php"** but it doesn't answer why this is happening very well.

**Comment by jdkoelsch** [comment]
We just ran into this issue too. We only encountered it when authenticated and viewing nodes of an Article content type. We reverted code, changed the theme back to Bartik. I even tried deleting and re-adding the content type with default fields, with no change -- same errors in the log.

Tried increasing php memory, destroyed VM, updated everything, rebuilt VM. Nothing helped. 

However,  changing http.response.debug_cacheability_headers to false fixed the issue.

Our dev environment, which is very similar to the DrupalVM setup, does not experience this issue. I tested against a copy of the dev db to verify.
